### ISSUE TITLE
Installed Add-on will not Enable

### ISSUE THREAD

**Issue by Davexx** [johnnygizmo/BulkAsset#1]
Hello,
I was attempting to try your add-on using Blender 3.5 on Windows 11.
In Blender: 
Edit > Preferences > Add-ons > Install... , chose "BulkAsset1.5.zip" 
"Check" Assets > Bulk Asset Tool and I receive the following error:

`Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 3.5\3.5\scripts\modules\addon_utils.py", line 369, in enable
    mod.register()
  File "C:\Users\Dave\AppData\Roaming\Blender Foundation\Blender\3.5\scripts\addons\BulkAsset\__init__.py", line 59, in register
    bpy.types.ASSETBROWSER_MT_asset.append(header_menu_func)
AttributeError: 'module' object has no attribute 'ASSETBROWSER_MT_asset'`

Therefore I was unable to install the add-on.
I also tried running Blender as an administrator, but that did not work either.
Is this a bug, or am I doing something wrong?

Thanks,
Davexx

**Comment by Davexx** [comment]
Using Windows 11 with Blender 3.5.0 and Bulk Assets 1.5
Tried a factory reset on Blender and I was able to get BulkAsset to install, so something in your Add-on is not working with one of the Blender settings.  
Managed to add paths back to my asset libraries located on an external drive, then attempted to "Change Author" on 4 files with the right-click menu.  Files did not update author name.  Tried the same with some other assets and they worked.  Some work and some don't.  Not sure why...
